content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -109,27 +108,25 @@ The following example demonstrates declaration of a Shader Resource Group. We ha
             return m_uniformColor.r;
         }
     };
-  
+```
 ### Scope
 Every SRG defines a scope and the SRG data must be qualified when accessed from out-of-SRG scope code.  
```

Comment:
- Improve flow.
- Be consistent with "**Example**"

```suggestion
Every SRG defines a scope. The SRG data must be qualified when accessed from outside of the SRG's scope in code.  

**Example**:
```

Reply:
Fixed.